css/form.css

```diff
@@ -1471,6 +1493,13 @@ form hr {
 
 .geolocation.btn {
   column-gap: 10px;
+  display: inline-block !important;
```

Comment:
Any way of not using `!important`?

Reply:
For that, I should remove `!important` styling from primary button's style, but I don't know how it will affect the other components.

Reply:
@armine-fliplet OK. We can keep this one then.